### PR TITLE
[MIG] purchase_order_revision: Migration to 11.0

### PR DESCRIPTION
**Pull request by Montxu**
cc @guadaltech

### CODE REVIEW

File: purchase_order_revision/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+{
+    "name": "Purchase order revisions",
+    "version": "11.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Agile Business Group,"
+              "Camptocamp,"
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "http://www.agilebg.com",
```
Comment (nikul-serpentcs) [inline purchase_order_revision/__manifest__.py]: @Montxu URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: purchase_order_revision/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2018 Ramon Bajona <ramon.bajona@guadaltech.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase order revisions",
+    "version": "11.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Agile Business Group,"
+              "Camptocamp,"
```
Comment (Jerther) [inline purchase_order_revision/__manifest__.py]: Add space after comma for better text display
image

File: purchase_order_revision/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright 2015 Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2018 Ramon Bajona <ramon.bajona@guadaltech.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase order revisions",
+    "version": "11.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Agile Business Group,"
```
Comment (Jerther) [inline purchase_order_revision/__manifest__.py]: Add space after comma for better text display
image

**Review by Jerther — APPROVED** [review]
Some minor changes in manifest would be nice, but other than that, code LGTM and works fine. Same for unit tests.

Side note: I also tried it on a 12.0 DB and it works as is, except for tests. Migration would be done quickly.

**Comment by Montxu** [comment]
@nikul-serpentcs can you check the changes to aprove?

**Review by nikul-serpentcs — APPROVED** [review]
Fix **Mr. Travis**, Otherwise code review LGTM :+1:

**Comment by nikul-serpentcs** [comment]
Fix **Mr. Travis**, Otherwise Code Review LGTM :+1:

**Comment by dreispt** [comment]
Travis is red.

**Comment by rousseldenis** [comment]
@Montxu Is this still applicable ?

**Pull request closed without merging**